google-cloud-firestore/.rubocop.yml

```diff
@@ -11,21 +11,24 @@ AllCops:
     - "conformance/**/*"
     - "test/**/*"
 
-Documentation:
+Style/Documentation:
   Enabled: false
 
+Metrics/AbcSize:
+  Exclude:
+    - "lib/google/cloud/firestore/query.rb"
 Metrics/ClassLength:
   Enabled: false
 Metrics/CyclomaticComplexity:
-  Max: 15
+  Max: 17
 Metrics/PerceivedComplexity:
-  Max: 15
+  Max: 17
 Naming/FileName:
   Exclude:
     - "lib/google-cloud-firestore.rb"
 Naming/RescuedExceptionsVariableName:
   Enabled: false
```

Comment:
Any way we can fix the code and remove this disabling?

Reply:
Done, thanks!